core_backend/tests/api/test_question_answer.py

```diff
@@ -619,6 +628,11 @@ async def async_fake_generate_speech(*args: Any, **kwargs: Any) -> str:
             assert "error" in json_response
             assert response.status_code == 500
 
```

Comment:
Since you setting the error message yourself you should assert it is the correct one